<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poruchenia</class>
 <widget class="QDialog" name="Poruchenia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поручения</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_add">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/add.png</normaloff>:/img/img/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButton_del">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/delete.png</normaloff>:/img/img/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_edit">
        <property name="text">
         <string>Редактировать или сменить статус</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/edit.png</normaloff>:/img/img/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButton_search">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/find.png</normaloff>:/img/img/find.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_update">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/refresh.png</normaloff>:/img/img/refresh.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_search">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Поиск</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="4,4,3,3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите статус</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Введите заголовок поручение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_status">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_title">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_find">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/findOk.png</normaloff>:/img/img/findOk.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Очистить поиск</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/img/img/clear.png</normaloff>:/img/img/clear.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_result">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 0, 255)</string>
     </property>
     <property name="text">
      <string>Поиск не дал результатов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTableView" name="tableView_poruchenia">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_naideno">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(85, 0, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/edit.png</normaloff>:/img/img/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать или сменить статус поручения</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/delete.png</normaloff>:/img/img/delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить выбранное поручение</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
